code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1480,6 +1528,34 @@
 	color = "#C8A5DC"
 	taste_description = "chemicals"
 
+/datum/reagent/hexamine
+	name = "Hexamine"
+	description = "A White crystalline compound useful for synthesizing other chemicals"
+	reagent_state = SOLID
+	color = "#E6FFFF"
+	taste_description = "chemicals"
+
+/datum/reagent/pentaerythritol
+	name = "Pentaerythritol"
+	description = "Slow down , it aint no spelling bee!"
```

Comment:
Typing error, extra space after "Slow down" -- also "ain't" should have an apostrophe.